### ISSUE TITLE
My deck not correct tracker and not upload replay After update for 1.6.0

### ISSUE THREAD

**Issue by utherbone** [HearthSim/Hearthstone-Deck-Tracker#3607]
[hdtreplay.zip](https://github.com/HearthSim/Hearthstone-Deck-Tracker/files/1836178/hdtreplay.zip)

## Bug report
After update from 1.5.14 for 1.6.0, in mode adventure "Knights of the Frozen Throne" my deck not correct tracker and not upload replay, message show: 

The uploaded file is not a valid replay.
{'server_ip': ['Enter a valid IPv4 or IPv6 address.'], 'server_version': ['A valid integer is required.']}

About the tracker problem, my cards seem all static, throughout the game.
It was a fun game, cool, I want to see again something that can be done about replay?
2 Last replays in the adventure mode I played, attached.
Thanks for the decktracker to exist, whenever I give it somehow I'll contribute.

**Comment by azeier** [comment]
Looks like HDT wasn't able to read data from the game correctly. I don't think we can do much about those specific replays, a lot of metadata is missing.

Try restarting HDT and Hearthstone and see if this problem persists.

**Comment by utherbone** [comment]
I restarted everything after playing some matches even with the problems. when the Tracker started accused a corrupted file PlayerDecks.xml he apologized and restored a backup (did not restore the replay he wanted, ok no problem) then I decided to rezet the Tracker putting default result returned to work correctly.

(Could it have been an instability in the game I did not feel it very fluid today.)
Thanks for the quick help, you are flash? Ok, I'm kidding
Continue with this work!
